app/models/truck.rb

```diff
@@ -0,0 +1,4 @@
+class Truck < ApplicationRecord
+  validates :license_plate, presence: true
+  validates :capacity, presence: true
```

Comment:
lets also set the enum status for trucks too!
`enum status: [ :active, :inactive, :servicing ]`
and make sure that we have a validation for the enum too, ensuring that anything outside our enums isnt accepted.

https://stackoverflow.com/questions/29780121/rails-test-validation-of-enum-fields

Reply:
@benrobinson-ms Are you sure it would be better to use `enums` as it raises an exception on init, not on a regular AR validation stage.

Reply:
not sure if i did the validation for this correctly

Reply:
@mmmmaximus all good.
My opinion that just think need to choose:

```
enum status: [ :active, :inactive, :servicing ]
```

OR

```
validates :status, inclusion: { in: Truck.statuses.keys }
```

As do the same thing.

Reply:
`Truck.statuses.keys` wouldnt return anything if we remove `enum status: [ :active, :inactive, :servicing ]` i think?